lib/licenses.nix

```diff
@@ -580,6 +580,11 @@ in mkLicense lset) ({
     fullName = "Licence Art Libre 1.3";
   };
 
+  laspy = {
+    fullName = "laspy license";
+    url = "https://github.com/laspy/laspy/blob/master/LICENSE.txt";
```

Comment:
What do you think about referring to a particular commit hash here? Ensures reproducibility :slightly_smiling_face:

Reply:
Probably overkill, the other licenses are broken too in that case :D

Reply:
True, well except for https://github.com/NixOS/nixpkgs/blob/b2ca5b4a2ac640aecac6b0053f5e17fdd31d8d06/lib/licenses.nix#L556

Reply:
this is just a BSD2 license, so no need to add it here.